src/axolotl/prompt_strategies/sharegpt.py

```diff
@@ -12,8 +12,8 @@
         system_template="<|im_start|>system\n{system_message}",
         system_message="You are a helpful assistant.",
         roles=["<|im_start|>user", "<|im_start|>assistant"],
-        sep_style=SeparatorStyle.CHATML,
```

Comment:
by removing this, the `Conversation` defaults to a `sep_style=SeparatorStyle.ADD_COLON_SINGLE` which results in colons getting added between the role and text.